planning/planning_index.adoc

```diff
@@ -246,6 +247,56 @@ directory between host and container using the _-v_ flag in your RUN label.
 === Security and User considerations
 
 ==== Passing credentials and secrets
+
+Storing sensitive data like credentials is a hot topic, especially because Docker does not provide a designated option for storing and passing secret values to containers.
+
+Currently, the most popular and a reasonable way to pass credentials and secrets in a container is specifying them as **environment variables** at container runtime. You as a consumer of such an image don't expose any of your sensitive data publicly.
```

Comment:
i would not say "reasonable" but doable if you know what are you doing... another way is to use tmpfs volume where you have the secrets stored (which is what kube is doing btw.)